src/workloads/execution/GraphLookup.yml

```diff
@@ -1,18 +1,25 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
 Description: |
-  This test exercises the behavior of $graphLookup against a sharded foreign collection.
+  This test exercises the behavior of $graphLookup where either the local or foreign collection is
+  sharded. When both collections are unsharded, the pipeline should be moved to the
+  GraphLookupInUnshardedEnvironment workload, so the query can be tested on replica sets as well. We
+  may want to compare results across the two files, so the loading stage (and some queries) should
+  be kept as similar as possible.
 
   The workload consists of the following phases:
     1. Creating empty sharded collections distributed across all shards in the cluster.
     2. Populating collections with data.
     3. Fsync.
-    4. Running $graphLookup into a sharded collection.
+    4. Running $graphLookups.
```

Comment:
Could you add some more detail to the 4th phase? Maybe state at a high level the cases we're testing (differing collections sharded, recursion depths, how many matches in the foreign collection for each one in the local coll, etc).